packages/babel-loader-8/index.js

```diff
@@ -0,0 +1,17 @@
+const { applyVariantToBabelConfig } = require('@embroider/core');
+module.exports = require('babel-loader').custom(babel => {
```

Comment:
A short description of the "fix" would be great. As it may not be obvious for future travelers.

Reply:
I added more to commit description